mpsl/include/mpsl_fem_config_common.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file mpsl_fem_config_common.h
+ *
+ * @defgroup mpsl_fem_config_common MPSL Front End Module Common Configuration parts
+ * @ingroup  mpsl_fem
+ *
+ * The MPSL Front End Module Common Configuration defines structures common for every supported Front End Module module.
+ * @{
+ */
+
+#ifndef MPSL_FEM_CONFIG_COMMON_H__
+#define MPSL_FEM_CONFIG_COMMON_H__
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Configuration parameters for pins that enable or disable (or both) either Power Amplifier (PA) or Low Noise Amplifier (LNA).
+ */
+typedef struct
+{
+    bool    enable;       /**< Enable toggling for this pin. */

```

Comment:
Why would someone configure a pin for PA / LNA frontend that could not be toggled?